Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:0.10.43
+
+MAINTAINER Germán Toro del Valle <german.torodelvalle@telefonica.com>
+
+RUN mkdir /github && mkdir /github/telefonicaid
+
+WORKDIR /github/telefonicaid
+RUN git clone https://github.com/telefonicaid/context-adapter.git
+
+WORKDIR  /github/telefonicaid/context-adapter
+RUN git fetch && git checkout master && npm install
+
+EXPOSE 9999
+
+WORKDIR /github/telefonicaid/context-adapter
+CMD ["npm", "start"]
```

Comment:
Yeah, I included an npm script in this very PR: https://github.com/telefonicaid/context-adapter/pull/73/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23 Thanks for noticing! ;)